bedrock/firefox/templates/firefox/features/password-manager.html

```diff
@@ -39,7 +39,7 @@ <h1 class="c-hero-title">{{ ftl('password-manager-password-manager') }}</h1>
     aspect_ratio='mzp-has-aspect-1-1',
     class='mzp-l-card-feature-right-half mzp-t-firefox',
   ) %}
-   <p>{{ ftl('password-manager-forget-the-reset', url='https://support.mozilla.org/kb/use-master-password-protect-stored-logins') }}</p>
+   <p>{{ ftl('password-manager-forget-the-reset-v2', fallback='password-manager-forget-the-reset', url='https://support.mozilla.org/kb/use-master-password-protect-stored-logins') }}</p>
```

Comment:
The SUMO URL has changed to `https://support.mozilla.org/kb/use-primary-password-protect-stored-logins`, but there's a redirect so this doesn't break. But we could update the URL while we're at it (optional).